src/controller/cve-id.controller/cve-id.controller.js

```diff
@@ -350,16 +310,16 @@ async function reserveCveId (req, res) {
   Object.keys(req.query).forEach(k => {
```

Comment:
Since we were able to iterate through the ctx.query object in another function like this above, if we're going to keep this code, I think I would prefer this to do the same to avoid touching the raw query after it has been parsed and put in the ctx object. I know we're not pulling from it but it would just be best to stay away from it.